packages/core/src/config.ts

```diff
@@ -1571,9 +1571,11 @@ export async function composeCreateRsbuildConfig(
     ...sharedRsbuildConfig
   } = rslibConfig;
 
-  if (!libConfigsArray) {
+  if (!Array.isArray(libConfigsArray) || libConfigsArray.length === 0) {
```

Comment:
```suggestion
  if (Array.isArray(libConfigsArray) && libConfigsArray.length > 0) {
```

I think this way more easy to understand.

Reply:
`!Array.isArray(libConfigsArray) || libConfigsArray.length === 0` is equal to `!(Array.isArray(libConfigsArray) && libConfigsArray.length > 0)`.

Reply:
不是数组或数组里没有元素 -> 报错

Reply:
Yes, identical. Just avoid using `!` if possible.

Reply:
Nevermind, quite the same.